react/Button/Button.less

```diff
@@ -111,3 +111,47 @@
 .fullWidth {
   width: 100%;
 }
+
```

Comment:
Does it make sense to have the group style in the same less file?  Seems like it's big enough that a file of its own in the `ButtonGroup` directory would be appropriate

Reply:
@mengtzu The issue is that we need to reference classes from both components, since there's a parent/child relationship.